php/Terminus/Models/Workflow.php

```diff
@@ -23,6 +23,9 @@ class Workflow extends TerminusModel {
    */
   private $user;
 
+  // @TODO: Make this configurable.
```

Comment:
You'll notice that `Workflow.php` has been removed from my PR yet this thread is not marked "outdated". /shrug